packages/lesmis/src/views/DashboardView.js

```diff
@@ -59,7 +59,14 @@ export const DashboardView = () => {
             </FlexRow>
             <img src="/images/school.png" alt="place" />
             <FlexRow>
-              <Typography>Development Partner Support</Typography>
+              <TitleContainer>
+                <Typography>Development Partner Support</Typography>
+              </TitleContainer>
+              {Object.entries(vitals)
+                .filter(([, value]) => value === true)
+                .map(([key]) => (
+                  <PartnerLogo code={key} />
+                ))}
```

Comment:
I want to make this smarter by having vitals return a `vitals.DevelopmentPartners` object, but that involves adjusting the report so I'd prefer to do that after I consolidate my branches